### PR TITLE
Add stubbed test for BZ-1847889

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
**Description:**
1. Added stubbed test for [BZ-1847889](https://bugzilla.redhat.com/show_bug.cgi?id=1847889)
2. Fix test `test_positive_check_hotfix_installed` by replacing exit code for warning with 78, Changes from https://projects.theforeman.org/issues/29699

**Test Results:**
```
$ pytest --ansible-host-pattern server --ansible-user=root --ansible-inventory testfm/inventory tests/test_health.py::test_positive_check_hotfix_installed -sq
2021-01-04 14:46:58,928 - testfm.log - INFO - Running preparation steps required to run the next scenarios
================================================================================
Check whether system has any non Red Hat repositories (e.g.: EPEL) enabled:
                                                                      [SKIPPED]
--------------------------------------------------------------------------------


Running ForemanMaintain::Scenario::FilteredScenario
================================================================================
Check to verify if any hotfix installed on system:
/ Checking for presence of hotfix(es). It may take some time to verify.
                                                                      [WARNING]

HOTFIX rpm(s) applied on this system:
hotfix-package-1.0.0-1.HOTFIX.noarch

Found 1 file(s) modified on this system.
/opt/theforeman/tfm/root/usr/share/gems/gems/fog-vsphere-3.4.0/lib/fog/vsphere/requests/compute/list_clusters.rb

*** WARNING: Before update make sure the updated packages contain the listed modifications
*** otherwise these fixes will be lost.
*** It is also recommended to backup the modified files prior update.
--------------------------------------------------------------------------------
Scenario [ForemanMaintain::Scenario::FilteredScenario] failed.

The following steps ended up in warning state:

  [check-hotfix-installed]

The steps in warning state itself might not mean there is an error,
but it should be reviewed to ensure the behavior is expected
.2021-01-04 14:47:16,647 - testfm.log - INFO - Running preparation steps required to run the next scenarios
================================================================================
Check if tooling for package locking is installed:                    [OK]
--------------------------------------------------------------------------------


Running unlocking of package versions
================================================================================
Unlock packages:                                                      [OK]
--------------------------------------------------------------------------------
2021-01-04 14:47:56,989 - testfm.log - INFO - Running preparation steps required to run the next scenarios
================================================================================
Check if tooling for package locking is installed:                    [OK]
--------------------------------------------------------------------------------


Running locking of package versions
================================================================================
Lock packages:                                                        [OK]
--------------------------------------------------------------------------------

1 passed in 279.19 seconds
```

### CODE REVIEW

**Review by vsedmik — COMMENTED** [review]
Just reconsider keeping the code in a stubbed test, if it's not to be automated.

File: tests/test_health.py [hunk 1]
```diff
@@ -743,3 +743,38 @@ def test_positive_check_env_proxy(ansible_module):
...
+        1. foreman-maintain should be installed.
+        2. Satellite instance configured with external DHCP like Infoblox,
+           which has `:use_provider: dhcp_infoblox` set in /etc/foreman-proxy/settings.d/dhcp.yml
+        3. Satellite instance which is DHCP enabled and has `:use_provider: dhcp_isc`
+           set in /etc/foreman-proxy/settings.d/dhcp.yml
+
+    :steps:
+        1. foreman-maintain health list | grep foreman-proxy-verify-dhcp-config-syntax
+        2. foreman-maintain health check --label foreman-proxy-verify-dhcp-config-syntax
+        3. Assert that EPEL repos are enabled on system.
```
[the remark this answers is not in the document]
Comment (Gauravtalreja1) [reply tests/test_health.py]: Good Catch, removed :)

**Review by vsedmik — APPROVED** [review]
LGTM

**Review by jameerpathan111 — APPROVED** [review]
LGTM

**Review by lvrtelov — COMMENTED** [review]
I'm wondering why keeping a code in stubbed test, which will never be executed. If it's not going to be automated. @jameerpathan111 I'm not really sure if there are any guidelines for this in testfm :)

**Comment by Gauravtalreja1** [comment]
> I'm wondering why keeping a code in stubbed test, which will never be executed. If it's not going to be automated. @jameerpathan111 I'm not really sure if there are any guidelines for this in testfm :)

I agree to not keep any code in stubbed tests that never will execute but was following existing stubbed tests in testfm to write this one, that's why kept that code earlier. please review @lvrtelov

**Comment by jameerpathan111** [comment]
> I'm wondering why keeping a code in stubbed test, which will never be executed. If it's not going to be automated. @jameerpathan111 I'm not really sure if there are any guidelines for this in testfm :)

@lvrtelov Nope, no guideline. yes, I guess we should follow same practice as rest of repos and remove this code.

**Comment by lvrtelov** [comment]
> > I'm wondering why keeping a code in stubbed test, which will never be executed. If it's not going to be automated. @jameerpathan111 I'm not really sure if there are any guidelines for this in testfm :)
> 
> I agree to not keep any code in stubbed tests that never will execute but was following existing stubbed tests in testfm to write this one, that's why kept that code earlier. please review @lvrtelov
Looks good to me now, ACK :)

**Review by lvrtelov — APPROVED** [review]

**Pull request merged**